src/partials/pages/_3d-modeling.html

```diff
@@ -49,18 +41,18 @@ <h2 class="methods">Способи створення моделі:</h2>
         По вже існуючому прототипу шляхом зняття розмірів
       </li>
     </ul>
-    <div class="background-modeling"></div>
+    <div class="methods__background-model"></div>
   </div>
 </section>
 
 <!-- 3D MODELS -->
-<section class="section section-models padding-bottom">
+<section class="section models padding-bottom">
```

Comment:
There must be no classes named padding-bottom - geometry must be given to BEM mix.
For this case it must be <section class="section models"> and class models must contain the padding-bottom class properties> instead of being the separate class.

Reply:
the same situation as with comment about button below 
such class contain in all pages (modeling, scanning, printing, order-page, main-page, contact page, etc) 

see please =)

what should we do with it? ))

Reply:
In such case, when the mistake is global and touches not only your page - ask me to decide, which you did right. :)

In this particular case you leave it and create the card in Trello, saying to: "rework current global .padding-bottom class to either placeholder (if property is universal and has the same value everywhere), or mixin (if property is universal, but has different values in use), or global BEM modificator (if property is used to the same tags, like h2 only and has the same value everywhere).

I don't give you a direct answer for purpose, to stimulate investigation. Though if that overburdens you - let me know and I'll give the direct command.